hint_glfw.go

```diff
@@ -7,10 +7,20 @@ import "github.com/go-gl/glfw/v3.1/glfw"
 type Hint int
 
 const (
-	AlphaBits = Hint(glfw.AlphaBits)
-	Samples   = Hint(glfw.Samples)
+	AlphaBits                            = Hint(glfw.AlphaBits)
+	Samples                              = Hint(glfw.Samples)
+	Depth                           Hint = 0x00021100 // it works on WebGL and is ignored on other environment
+	Stencil                         Hint = 0x00021101 // it works on WebGL and is ignored on other environment
```

Comment:
It just occurred to me that GLFW already has hints for DepthBits and StencilBits, similar to AlphaBits. We should use those.

https://godoc.org/github.com/go-gl/glfw/v3.1/glfw#DepthBits

I can merge this PR and apply the fix on top.